public_html/wp-content/plugins/pattern-creator/src/plugins/main-dashboard-button.js

```diff
@@ -0,0 +1,11 @@
+import { registerPlugin } from '@wordpress/plugins';
+/* eslint-disable-next-line @wordpress/no-unsafe-wp-apis -- Experimental is OK. */
+import { __experimentalMainDashboardButton as MainDashboardButton } from '@wordpress/edit-post';
+
+registerPlugin( 'main-dashboard-button-test', {
+	render: () => (
+		<MainDashboardButton>
+			<a className="main-dashboard-button" href="http://wordpress.org">{ '←' }</a>
```

Comment:
Ideally this will go back to the previous page, which might be the homepage, or it might be a pattern details page. That can happen in a followup PR.

Reply:
Yeah, I was going to tie into `history.back()` but I'm still not certain I understand all ways into the editor.